omod/src/main/java/org/openmrs/module/fhir2/providers/MedicationFhirResourceProvider.java

```diff
@@ -47,4 +52,14 @@ public Medication getMedicationByUuid(@IdParam @NotNull IdType id) {
 		}
 		return medication;
 	}
+	
+	@Search
+	@SuppressWarnings("unused")
+	public Bundle searchForMedication(@OptionalParam(name = Medication.SP_CODE) TokenOrListParam code,
+	        @OptionalParam(name = Medication.SP_FORM) TokenOrListParam dosageForm,
+	        @OptionalParam(name = Medication.SP_INGREDIENT_CODE) TokenOrListParam ingredientCode,
```

Comment:
Can you remove this parameter from the search? That way it won't end up being specified as something we support. You can change the value to be null in the call below so we don't have to change the rest of the API.

Reply:
done